features/transition-behavior.yml

```diff
@@ -2,3 +2,6 @@ name: transition-behavior
 description: "The `transition-behavior: allow-discrete` CSS declaration allows transitions for properties whose animation behavior is discrete. Such properties can't be interpolated and swap from their start value to the end value at 50%."
 spec: https://drafts.csswg.org/css-transitions-2/#transition-behavior-property
 group: transitions
+compat_features:
+  - css.properties.transition-behavior
+  - css.properties.transition.transition-behavior
```

Comment:
Adding this currently demotes the feature status from baseline:low to baseline:false.
Is your plan to wait for the BCD bug you mentioned to get resolved before merging this? (So that the baseline status remains the same).

Reply:
I merged v5.6.12 today.